docs/design/zns-pivot/ZCF-2133-zns-pivot-detailed-design.md

```
@@ -0,0 +1,2006 @@
+private boolean shouldSync(String scope, String... targets) {
+    if (scope == null || "ALL".equalsIgnoreCase(scope)) return true;
+    for (String t : targets) {
+        if (t.equalsIgnoreCase(scope)) return true;
+    }
+    return false;
+}
+```
+
+> **scope 值映射**：`FABRIC` → TZ+TN；`TENANT` → Tenant+TenantRouter。
+> `shouldSync(scope, "TZ", "FABRIC")` 对 scope=FABRIC 返回 true，对 scope=TENANT 返回 false。
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)